Packs/CrowdStrikeFalcon/IncidentFields/incidentfield-Grand_Parent_Process.json

```diff
@@ -0,0 +1,30 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "grandparentprocess",
```

Comment:
please also add alias to the corealertfields pack for this new incident field

Reply:
also, I think we should add it to the common pack because this name is too generic